src/types/types.js

```diff
@@ -796,6 +799,9 @@ export type EdgeCurrencyWallet = {
   +signTx: (tx: EdgeTransaction) => Promise<EdgeTransaction>,
   +sweepPrivateKeys: (edgeSpendInfo: EdgeSpendInfo) => Promise<EdgeTransaction>,
 
+  // Accelerating:
+  +accelerate: (tx: EdgeTransaction) => Promise<EdgeTransaction | null>,
```

Comment:
I wonder if we should return `undefined` instead of `null` if this doesn't apply. We don't use `null` in very many places within the app.

Reply:
I always followed the idea that `undefined` means something wasn't present or available, whereas `null` is an actual return value with meaning which is like a null object, or an empty object (`typeof null === 'object'` afterall). The idea is that null packs a little different semantic then `undefined` and in this case it means the implementation must be forced to return `null` explicitly (removes the risk of implementation accidentally not covering a codepath and therefore an omition of a return statement is equivalent to `return undefined`).

Reply:
Sure, makes sense.